src/App.js

```diff
@@ -62,8 +59,9 @@ class App extends Component {
   }
 
   render () {
-    const {interactivePhase, datasetName, query, queryType} = this.state
-    const dataset = datasetName ? `https://data.world/${datasetName}` : null
+    const {interactivePhase} = this.state
+    const {dataset_name, query, queryType} = this.connector.params
```

Comment:
This change is not related to the "The user has not yet visited the url" error. This is here to improve the user experience for folks editing an existing connection. Previously, we would restore state from the URL. However, if users use the form and tweak/change the dataset or query params, then we need to restore state from `tableau.connData` instead.